src/partials/projects/projects.html

```diff
@@ -1 +1,166 @@
-<section class="projects">PROJECTS</section>
+<section class="projects">
+    <div class="swiper-container container">
+        <div class="projects-header">
+            <h2>PROJECTS</h2>
+          </div>
+      <div class="swiper-wrapper">
+        
+        <div class="swiper-slide">
+          <ul class="project-list">
+
+
+            <li class="project-list-item">
+                <div class="project-card">
+                  <div class="left-part">
+                    <div class="technologies">
+                      <div class="technology">#react</div>
+                      <div class="technology">#js</div>
+                      <div class="technology">#node js</div>
+                      <div class="technology">#git</div>
+                    </div>
+                    <h3>Programming Across Borders: Ideas, Technologies, Innovations</h3>
+                    <button class="project-btn" onclick="openInNewTab('https://https://github.com/project1')">See project</button>
```

Comment:
This line outputs an error to the console.
I suggest changing the button to a link
```html
                <a
                  class="project-btn"
                  href="https://github.com/gudzsv/project-syntax-samurais-02"
                  target="_blank"
                >
                  See project
                </a>
                  See project
                </a>
```